src/main/java/io/github/thebusybiscuit/hotbarpets/HotbarPet.java

```diff
@@ -19,4 +29,26 @@ public HotbarPet(Category category, SlimefunItemStack item, ItemStack food, Item
 	public ItemStack getFavouriteFood() {
 		return this.food;
 	}
+
+	/**
+	 * Tries to consume the pets favourite food, if this was successful it will return true. Otherwise false.
+	 * If it fails to consume food it will send a message to the player, this is on a delay as to not spam the player.
+	 * The default delay is {@link #MESSAGE_DELAY}.
+	 *
+	 * @param player  The {@link Player} who owns this pet
+	 * @return If the food consumption was successful
+	 */
+	public boolean tryToConsumeFood(Player player) {
+		if (!player.getInventory().containsAtLeast(getFavouriteFood(), 1)) {
+			if (messageDelay.getOrDefault(player.getUniqueId(), 0L) <= System.currentTimeMillis()) {
+				player.sendMessage(ChatColors.color("&9Your " + getItemName()
+						+ " &9would have helped you if you did not neglect it by not feeding it :("));
+				messageDelay.put(player.getUniqueId(), System.currentTimeMillis() + MESSAGE_DELAY);
```

Comment:
I dont really see a need for the message delay.
But this never gets cleared over time, so on a busy server this Map will just accumulate over time...